src/components/annotations3d/attributes.js

```diff
@@ -75,13 +75,13 @@ module.exports = {
     arrowwidth: annAtts.arrowwidth,
     standoff: annAtts.standoff,
     hovertext: annAtts.hovertext,
-    hoverlabel: annAtts.hoverlabel
+    hoverlabel: annAtts.hoverlabel,
+    captureevents: annAtts.captureevents
```

Comment:
what about the `plotly_clickannotation` event? Looks like that might need a `sceneId` if there is one? Which brings up the point that for extensibility to other non-cartesian subplots perhaps it should really be called `subplotId`...